tools/pdp-metadata/queries/products.graphql.js

```diff
@@ -6,11 +6,32 @@ export default `query productSearch($currentPage: Int = 1) {
         sku
         name
         urlKey
+        url
         shortDescription
         description
         metaDescription
         metaKeyword
         metaTitle
+        inStock
```

Comment:
We can likely use 
```
images(roles: "image") {
   url
}
```
for retrieving the primary product image. That should be more consistent with what the PDP dropin is doing.